### DISCUSSION TITLE
修改节点shape

### DISCUSSION THREAD

**Discussion by avrinfly** [antvis/X6#2011] [Q&A] [answered]
我新建了一个shape为rect的节点，我想修改这个节点的shape，只能先删掉这个节点再在原位置新建一个节点吗？有没有更简洁的方法，求大佬的指教~

**Comment by NewByVector** [comment, marked as answer]
不能动态修改 shape 的，但是可以修改 markup.

**Reply by avrinfly** [reply]
OK，我试试~

**Reply by gloriousknight** [reply]
> OK，我试试~

我也遇到了这个问题，现在还在试，请问大佬有解决方案了吗？
